### DISCUSSION TITLE
Features

### DISCUSSION THREAD

**Discussion by collerek** [collerek/cerebral-centaurs#9] [Ideas]
Let's discuss the features:

* General
* GUI
  *  Different tools available:
      * Eraser
      * Rectangle
      * Line
      * Spray
      * Circle /  Elipse
* Business Logic
* Bugs

**Comment by Palmirka** [comment]
- Add text field to tools
- Let user see changes added before he joined

**Comment by collerek** [comment]
* user should be able to create the room and get the ID
* new joiners can join a room or create a new one
* user can set username that has to be unique Ina room and used as client_id
* user can set avatar?
